index.js

```diff
@@ -62,4 +63,14 @@ app.get("*", function(req, res) {
   res.render("error");
 });
 
-app.listen(port);
+app.listen(port, function() {
+  if (app.get("env") === "development") {
+    browserSync({
+      files: ["static/**/*.css", "views/**/*.handlebars"],
+      online: false,
+      port: port + 1,
+      proxy: "localhost:" + port,
+      ui: false
+    });
+  }
+});
```

Comment:
When I look at the documentation of browser-sync, it is used as a command-line tool (in scripts).
If so, do we still need nodemon?
And why does port need to be changed? :)